### DISCUSSION TITLE
Help！ Source Code run error

### DISCUSSION THREAD

**Discussion by Jengro777** [toeverything/AFFiNE#448] [General]
os: Debian11
node: 17
![1111111111111111111111111111111111111](https://user-images.githubusercontent.com/35259040/191474233-e686e4ab-18c2-4246-9494-47b2ef0d580b.png)

**Comment by lawvs** [comment]
Sorry for any inconvenience, we have adjusted webpack's watch strategy, can you update the code and try again?

For details, see https://github.com/toeverything/AFFiNE/pull/455

**Reply by Jengro777** [reply]
> Sorry for any inconvenience, we have adjusted webpack's watch strategy, can you update the code and try again?
> 
> For details, see #455

ok! I'm up and running now, but my server is Linux, serving access outside. I've tried searching for a long time and can't find a place where I can modify the listening ip address, can you help me?
![image](https://user-images.githubusercontent.com/35259040/191903459-02e62bc6-8141-4a46-971b-ce89e83ea0aa.png)

**Reply by ShortCipher5** [reply]
@Avey777, you can use the host flag `--host=0.0.0.0`.

With the default command that might look something like this `pnpm start --host=0.0.0.0`

**Reply by Jengro777** [reply]
OK  It's great!

**Reply by Jengro777** [reply]
When running from source, I don't know how to set up local storage. Although docker works, I still want to know how to configure server-side storage while allowing source code.
